python/tskit/tables.py

```diff
@@ -1581,7 +1581,7 @@ def simplify(
             filter_zero_mutation_sites=None,  # Deprecated alias for filter_sites
             reduce_to_site_topology=False,
             filter_populations=True, filter_individuals=True, filter_sites=True,
-            keep_unary=False):
+            record_provenance=True, keep_unary=False):
```

Comment:
Adding `record_provenance` as the penultimate arg (not simply add to the list), so that it matches the argument order in ts.simplify()